guides/tutorial/start-tutorial.md

```diff
@@ -102,7 +102,7 @@ defmodule AuthMe.UserManager.Guardian do
   end
 
   def resource_from_claims(%{"sub" => id}) do
-    case UserManager.get_user(id) do
+    case UserManager.get_user!(id) do
```

Comment:
We use `!` when the function raises an error in Elixir. In this case, we are not using `rescue`.

I am curious why you suggested the change since it may be the case that we need to fix the tutorial somewhere else instead.

Reply:
sorry I'm quite new to elixir but changed to use `!` because the generator only created `get_user!`, otherwise function is undefined

Reply:
Ahhhh I see why you want it, in that case, changing it to `!` will no work thou.

Are you talking about Phoenix generator?

Reply:
So it seems that phoenix generators removed the non-error returning version of get_uses, so we will have to use the get_user! method now, but we should also update the case statement to reflect this 

```suggestion
    case UserManager.get_user!(id) do
          {:error, err} -> {:error, :resource_not_found}
          {:ok, user} -> {:ok, user}
```